mqtt/mqtt-bridge/src/persist/waking_state/ring_buffer/mod.rs

```diff
@@ -865,6 +870,85 @@ mod tests {
         }
     }
 
+    #[test]
+    fn it_inits_ok_with_previous_data_and_write_pointer_is_reaches_read_pointer_after_read() {
+        let result = tempfile::NamedTempFile::new();
+        assert_matches!(result, Ok(_));
```

Comment:
I'm not sure why do you need this? why not just use `.expect("...")` or `.unwrap()` directly? I believe asserts are better for actual test results you want to verify, not for intermediate/setup code.

Reply:
Yeah will change :)